app/src/organisms/FirmwareUpdateModal/UpdateInProgressModal.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { css } from 'styled-components'
+import {
+  ALIGN_CENTER,
+  BORDERS,
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  SPACING,
+  TYPOGRAPHY,
+} from '@opentrons/components'
+import { ProgressBar } from '../../atoms/ProgressBar'
+import { StyledText } from '../../atoms/text'
+import { Modal } from '../../molecules/Modal'
+
+interface UpdateInProgressModalProps {
+  percentComplete: number
+}
+
+const OUTER_STYLES = css`
+  background: ${COLORS.medGreyEnabled};
+  width: 100%;
+`
+
+export function UpdateInProgressModal(
+  props: UpdateInProgressModalProps
+): JSX.Element {
+  const { percentComplete } = props
+  const { i18n, t } = useTranslation('firmware_update')
+
+  return (
+    <Modal>
+      <Flex
+        height="12.5rem"
```

Comment:
![Screenshot 2023-07-18 at 12 56 21 PM](https://github.com/Opentrons/opentrons/assets/474225/95e7c85b-91bb-4f90-9441-2c3c394a5939)


```suggestion
        height="17.25rem"
```